data/out/tables/sample1.csv/sample1.csv

```diff
@@ -0,0 +1,5 @@
+"someText","","",""
```

Comment:
nemělo by to přidat header? Jestli jsem to pochopil, tak processor headers je deprecated nebo co a tohle ho nějak nahrazuje? https://developers.keboola.com/extend/component/processors/#extractor-example (4. blok kódu od nadpisu)

Reply:
@odinuv tady to zmizelo tím smazáním, ale dotaz platí... ;)

Reply:
Aha, už jsem někde v panelech našel odněkud otevřené https://github.com/keboola/processor-headers/issues/1, kde se to asi řeší ;)

Reply:
Tedy tohle bude řešit create manifest potom, co se to prožene tímhle a tím pádem to bude mít správný počet sloupců všude.

Reply:
naopak nejdriv se zavola createmanifest - ten vezme sloupecky z prvniho radku a pripadne se udela headerless csv skiplines, tohle predpoklada ze csv je headerless a zadny header tedy nepridava